tests/test_opentime.py

```diff
@@ -649,20 +664,20 @@ def test_passing_ndf_tc_at_df_rate(self):
         tc1 = otio.opentime.to_timecode(
             otio.opentime.RationalTime(frames, 29.97)
         )
-        self.assertEqual(tc1, DF_TC)
+        assert (tc1 == DF_TC)
 
         tc2 = otio.opentime.to_timecode(
             otio.opentime.RationalTime(frames, 29.97),
             29.97,
             drop_frame=False
         )
-        self.assertEqual(tc2, NDF_TC)
+        assert (tc2 == NDF_TC)
 
         t1 = otio.opentime.from_timecode(DF_TC, 29.97)
-        self.assertEqual(t1.value, frames)
+        assert (t1.value, frames)
```

Comment:
This is asserting a two-element tuple, which will always pass. It should likely be:
`assert t1.value == frames`